native/dgetrf.go

```diff
@@ -29,7 +29,7 @@ func (impl Implementation) Dgetrf(m, n int, a []float64, lda int, ipiv []int) (o
 		panic(badIpiv)
 	}
 	if m == 0 || n == 0 {
-		return
+		return false
```

Comment:
Good point. I hadn't thought about that. Hasn't caused any issues, so LGTM. If it breaks something we can reassess.